boxen/assets/scrapli_platforms/checkpoint_cloudguard.yaml

```diff
@@ -0,0 +1,26 @@
+---
+default:
+  driver-type: "network"
+  privilege-levels:
+    configuration:
+      name: "configuration"
+      pattern: '(?im)^[\w.\-@/:]{1,63}>\s*$'
+      previous-priv: ""
+      deescalate: ""
+      escalate: ""
+      escalate-auth: false
+      escalate-prompt:
```

Comment:
lazy way of adding a single privlevel platform. It works, but I am not sure if it is better be done otherwise?

Reply:
I left it as is (with a single config priv level), since there is a bash priv level that someone might need to add later on